### DISCUSSION TITLE
Get Unity games working!

### DISCUSSION THREAD

**Discussion by iniMerchant64** [skyline-emu/skyline#426] [Game Requests]
A lot of people want stuff like Gungeon, Hollow Knight, Ori or even Cuphead working, including me

**Comment by sauradipdebnath** [comment]
Is ori popular

**Reply by iniMerchant64** [reply]
it is quite a bit, saw at least a dozen of people asking for it

**Reply by adageeber** [reply]
Would love to see hollow knight working. Would blow the minds of my cousins next time they visit heh.

**Reply by adageeber** [reply]
I own the game on PC, but I want to just play on the go sometimes

**Reply by Rimurudt** [reply]
You own the game on pc but don't own on switch so how would you get the game to play on the go?

**Reply by adageeber** [reply]
Not everyone has the money for both devices, you understand that right?
And I don't think I want to even buy a switch if all I'm going to play on it is Hollow Knight.

**Reply by adageeber** [reply]
That's just wasteful at that point

**Reply by iniMerchant64** [reply]
Buddy in order to legally emulate stuff you need to own the console. Emulation isnt meant to be an alternative of any sorts.

**Reply by PixelyIon** [reply]
@adageeber Sadly, that's not a use-case we can support as the only legal way to dump games is via a hacked Switch. It would be piracy if you downloaded the game from any other source.

**Reply by adageeber** [reply]
Alright then, I understand; I won't be stubborn about it or anything like that. Still, it's nice to know that this stuff can be done on a device that we already own.

**Reply by adageeber** [reply]
Legal stuff flies over my head, but I appreciate your integrity

**Reply by PixelyIon** [reply]
Understandable, it's a use-case we would like to support if it was possible but it just isn't and if we don't follow the law then litigation by Nintendo among other entities would take us down and rightfully so.

**Comment by Owd31222** [comment]
I think Unity is a good start

**Comment by Luckypro23** [comment]
How much time it Takes to make hollow knight Playable ?

**Reply by PixelyIon** [reply]
We don't provide any ETAs.
